.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   build-and-test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v2.3.4
```

Comment:
:+1: probably best to stay up to date (or maybe this is reverting a reversion from elsewhere?)

Reply:
I will change this back to 2